skimage/filters/_gaussian.py

```diff
@@ -11,6 +11,14 @@
 
 
 def _convert_input(image, preserve_range):
+    """
+    Parameters
+    ----------
+    preserve_range : bool, optional
+    Whether to keep the original range of values. Otherwise, the input
+    image is converted according to the conventions of `img_as_float`.
+    See more http://scikit-image.org/docs/dev/user_guide/data_types.html.
```

Comment:
@Jathrone 

Either "See URL for more" or "See more at URL". Sorry. =P Probably the former is nicer for line length.